### ISSUE TITLE
Offscreen with iPadOS 13 split screen multitasking

### ISSUE THREAD

**Issue by cliffjoyce** [vsanthanam/VSAlert#48]
**Describe the bug**
When VSAlertController appears in an app that is sharing the iPadOS screen with another app, then the alert is partially or fully offscreen.  This happens regardless of which VSAlertControllerStyle is specified.

**To Reproduce**
1. Run an app that uses VSAlertController on iPadOS 13.
2. Swipe up from bottom of screen to reveal iPad dock.
3. Drag an app icon from the dock to the right of the opened app-- to split the screen.
4. Trigger a VSAlertController alert from the first app
5. The VSAlertController appears partially or fully offscreen.

**Expected behavior**
The VSAlertController should be fully visible and sized/centered to fully fit within the app's UIWindow. 

**Screenshots**
See the "VSAlertController issue.png"

**Smartphone (please complete the following information):**
 - Device: any iPad
 - OS: iPadOS 13

![VSAlertController issue](https://user-images.githubusercontent.com/266027/77360389-806dc700-6d0a-11ea-8426-0f6179c7df4a.png)

**Comment by cliffjoyce** [comment]
I did find two temporary workarounds: 

(1) Before calling presentViewController on the VSAlertController, I set its frame to my app's UIWindow bounds.

(2) When using the VSAlertControllerStyleWalkthroughAlert and my app's UIWindow bounds are less than 500px, I set VSAlertControllerStyle to VSAlertControllerStyleAlert instead of VSAlertControllerStyleWalkthroughAlert.

Nevertheless, I wish VSAlertController would auto-adjust its frame to fit within the host app's UIWindow's bounds since VSAlertController is meant to be a drop-in replacement for UIAlertController.
